CHANGELOG.md

```diff
@@ -23,11 +23,21 @@ All notable changes to this project are documented in this file.
  - Added `slint::Window::scale_factor()` as getter to read the system device pixel ratio.
  - Added support for `slint::Window::set_size` and `slint::Window::size` to set and get the
    size of the window on the screen.
- - Added `slint::Window::dispatch_pointer_event` and `slint::PointerEvent` to be able to manually
+ - Added `slint::Window::dispatch_event` and `slint::WindowEvent` to be able to manually
    send a mouse or touch event to a window.
  - Added `animation-tick()`
  - `SharedString` implements `std::fmt::Write` and added `slint::format!`
  - `Image` can now be rotated with the `rotation-*` properties
+ - Use docking widgets and integration of slint-lsp into the Online Code Editor
+
+
+### Fixed
+ - Fixed Ctrl+Backspace/Ctrl+Del not deleting words in text input elements.
+ - Resizing of preview window from our editor extension
```

Comment:
```suggestion
 - Resizing of live-preview window in the IDE integrations.
```

(don't need to mention that it is "ours" as ownership is not relevant; Re-use the term "live-preview" that we've used previously; same for IDE integrations)